### PR TITLE
Add easy install instructions

### PR DESCRIPTION
**Pull request by danielbayley**
See https://github.com/danielbayley/homebrew-alfred/commit/9f4b02f977a90c866a5c6dc21aaae9b885635e9e

### CODE REVIEW

**Comment by sindresorhus** [comment]
I don't see how this is easier. The existing solution is literally just 2 clicks.

**Comment by danielbayley** [comment]
Sure, in isolation it might seem unnecessary… But manually setting up so many of these little tasks quickly adds up to a nightmare! (I recently got a new machine, so having to go through this pain again… hence my motivation.) The point is this allows installing Alfred themes as well as workflows… They can now be added to a `Brewfile` (as part of a dotfiles repo), and installed along with everything else, using `brew bundle`:
~~~ rb
# Brewfile

# …

# Alfred
tap 'danielbayley/alfred'
cask 'alfred-things'
cask 'alfred-evernote'
cask 'alfred-numi'
cask 'alfred-finder-new-item'
cask 'alfred-theme-dracula'
cask 'alfred-theme-simple' #-dark or -light

# …
~~~

Manually navigating to this repo in the first place is another manual step in this scenario…

@sindresorhus Maybe this benefit should be better emphasised, rather than the current instructions in this PR?

File: readme.md [hunk 1]
```diff
@@ -13,6 +13,12 @@
 
 *You need the [Powerpack](https://www.alfredapp.com/powerpack/) for custom themes.*
 
+Also, you can easily install both, or either theme with [Homebrew](https://brew.sh)[-alfred](https://github.com/danielbayley/homebrew-alfred):
+
+``` sh
+brew tap danielbayley/alfred
+brew cask install alfred-theme-simple #-dark or -light
```
Comment (sindresorhus) [inline readme.md:20]: Would be better to show the full command for both dark and light version.

File: readme.md [hunk 2]
```diff
@@ -13,6 +13,12 @@
 
 *You need the [Powerpack](https://www.alfredapp.com/powerpack/) for custom themes.*
 
+Also, you can easily install both, or either theme with [Homebrew](https://brew.sh)[-alfred](https://github.com/danielbayley/homebrew-alfred):
+
+``` sh
```
Comment (sindresorhus) [inline readme.md:18]:
```suggestion
```sh
```

**Comment by sindresorhus** [comment]
I don't think you need to show the benefits here, but you should definitely make it clearer in the https://github.com/danielbayley/homebrew-alfred readme.

**Comment by sindresorhus** [comment]
I'm ok with merging this.

**Comment by danielbayley** [comment]
> but you should definitely make it clearer in the https://github.com/danielbayley/homebrew-alfred readme.

Will do! I have many changes to push on there…

File: readme.md (hunk 2 above)
Comment (danielbayley) [reply readme.md]: Done.

**Comment by sindresorhus** [comment]
There's an update: https://github.com/sindresorhus/alfred-simple/commit/d4e597a15944feee58ffdc01011f67031f0225b6

Can you also update the install instructions in the readme?

**Comment by sindresorhus** [comment]
// @danielbayley

**Comment by danielbayley** [comment]
> There's an update: [d4e597a](https://github.com/sindresorhus/alfred-simple/commit/d4e597a15944feee58ffdc01011f67031f0225b6)
> 
> Can you also update the install instructions in the readme?

Sure!

**Comment by danielbayley** [comment]
> Can you also update the install instructions in the readme?

@sindresorhus Actually, the instructions are exactly the same… Are the screenshots not up to date?

**Comment by sindresorhus** [comment]
I don't remember why I asked it. It's been a while.

**Comment by danielbayley** [comment]
@sindresorhus No worries, the install instructions should never need to be updated as they remain exactly the same as far as the user is concerned… Only the [preview links](https://github.com/sindresorhus/alfred-simple/commit/a5b2304f3d7422c3c9bee660ba4889d05b02fd7d#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15) need to be reflected in the Casks when there is an update.

**Pull request merged**